share/tasks.py

```diff
@@ -208,7 +208,7 @@ def do_run(self, *args, **kwargs):
                 cg = ChangeGraph(self.normalized.data['@graph'], namespace=self.normalized.source.username)
                 cg.process()
                 cs = ChangeSet.objects.from_graph(cg, self.normalized.id)
-                if cs and (self.source.is_robot or self.source.is_trusted):
+                if cs and (self.source.is_robot or self.source.is_trusted or self.source.username == settings.APPLICATION_USERNAME):
```

Comment:
That seems to make sense... Who do we trust more than the system itself?

I'm not sure the best way to do it, though. Would that be a migration?